meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/qemuarm64/lmp.cfg

```diff
@@ -11,3 +11,4 @@ CONFIG_PREBOOT="virtio scan; env load"
 # CONFIG_ENV_IS_IN_FLASH is not set
 # CONFIG_MTD is not set
 # CONFIG_MTD_NOR_FLASH is not set
+CONFIG_BOOTSTD=y
```

Comment:
Do we actually need it?

Reply:
It fixes building. Let me check what depends on it.

Reply:
it fixes this
```
aarch64-linux-gnu-ld.bfd: boot/bootmeth_efi_mgr.o: in function `efi_mgr_check':
/home/cryo/work/u-boot/boot/bootmeth_efi_mgr.c:39: undefined reference to `bootflow_iter_check_system'
```